
console.log('this is an failing test')

require('assert').ok(false)